### ISSUE TITLE
Amazing work！Why can't I find this article online, can you upload a PDF file？

### ISSUE THREAD

**Issue by zxt6174** [songxt3/OST#3]

**Comment by songxt3** [comment]
Thanks for your attention. The conference proceedings of IJCAI are not published yet. Maybe next month will be available. 
PS. If we decide to upload the Arxiv version, I will also update this issue.

**Comment by zxt6174** [comment]
Thanks,！hope to get an arxiv version sooner！

**Comment by songxt3** [comment]
The papar is avialiable now at: https://www.ijcai.org/proceedings/2024/348

**Comment by zxt6174** [comment]
Thank you！I've successfully reproduced your code, but when I read the code, I see that the position embedding  doesn't seem to be being used. Equation (2) in your paper doesn't seem to be reflected in the code. Am I missing any details? Or is the spiking position embedding  special?
Hope to answer my confusion.

**Comment by songxt3** [comment]
Plz check cifar/model.py, Line 239 "x = x_1 + x_feat". This is the imply of the Eq.(2), i.e., the position embedding.

**Comment by zxt6174** [comment]
Ok，Thanks！Another question.  In "class Multi_Threshold_Acc(nn.Module)",  the output "x" after additive operation,  Its an integer valued vector ，but  a spike train.  Right?

**Comment by zxt6174** [comment]
Ok，Thanks！Another question. In "class Multi_Threshold_Acc(nn.Module)", the output "x" after additive operation, Its an integer valued vector ，but not a spike train. Right?

**Comment by songxt3** [comment]
Yes, the output of the "class Multi_Threshold_Acc(nn.Module)" part represents the intensity of the spikes.

**Comment by zxt6174** [comment]
Because you used this activation function for the "q""k" matrix in the attention module. This doesn't mean that, the elements in the “q”"k" matrix are floating-point numbers, and there is a floating-point multiplication operation .

**Comment by songxt3** [comment]
Only the calculations of 'Yt' would cause the situation you mentioned, and others are the spiking gate mechanism as we mentioned in the paper.
